c2corg_ui/static/partials/advancedsearch.html

```diff
@@ -4,5 +4,6 @@
     <app-card app-card-doc="::doc" ng-mouseenter="searchCtrl.onMouseEnter(doc.document_id)"
               ng-mouseleave="searchCtrl.onMouseLeave(doc.document_id)"></app-card>
   </div>
+  <div class="list-item" ng-repeat="n in [].constructor(5) track by $index"></div>
```

Comment:
To be honest, I am not fluent with angular yet, so this is a solution I found on the web after several unsuccessful attempts. If it is working, I'll do it. Does it?